### DISCUSSION TITLE
Duration column formatting in playlist like foobar2000?

### DISCUSSION THREAD

**Discussion by Samueru-sama** [DeaDBeeF-Player/deadbeef#2893] [Q&A] [answered]
In foobar2000 the duration column updates when you are playing a selected track to show the current second and total length of the track, in deadbeef it only shows the total duration without the current playing second.

For example if a track has a length of 2 minutes, it shows as 2:00 in both foobar2000 and deadbeef.

However if I start playing that track, in foobar2000 that will update to show 0:01/2:00 and that 0:01 will start updating every second along with the playing music. In deadbeef this doesn't happen.

Using a custom column with the format %playback_time% %length% almost does the trick, however it shows 0:01 2:00 instead of 0:01/2:00 when playing, I can't simply do %playback_time% / %length%  because the ''/'' would show in every single track that is not playing as /2:00 in this example, how can I fix this?

Also what's the title format for the default playing column in deadbeef? That shows a play icon when playing a track, I would love to know the tittle format to incorporate that to what I'm trying to do.

**Comment by Oleksiy-Yakovenko** [comment, marked as answer]
In deadbeef, this is something that's not used by default, but you can enable the same behavior by using this title formatting script for the Duration column: `[%playback_time% / ]%length%`

<img width="429" alt="image" src="https://user-images.githubusercontent.com/1895698/202922548-1639e629-d404-4d38-be0e-a3aa63899044.png">
<img width="135" alt="image" src="https://user-images.githubusercontent.com/1895698/202922558-025a139d-91d3-4574-9f5d-d15454933c97.png">

**Reply by Samueru-sama** [reply]
Thanks a lot, that was exactly what I needed.

**Comment by Oleksiy-Yakovenko** [comment]
> Also what's the title format for the default playing column in deadbeef? That shows a play icon when playing a track, I would love to know the tittle format to incorporate that to what I'm trying to do.

This column does not use title format -- it's a special column type, and it can't be done using title formatting.
